### ISSUE TITLE
Exclude option

### ISSUE THREAD

**Issue by dziamid** [Updater/protractor-console#5]
Would be nice to be able to exclude known issues from output, as in original protractor console plugin. For example:
 {
    package: 'protractor-console',
    logLevels: ['severe, warning'],
    exclude: ["some nasty bug or warning"]
  }

**Comment by dinvlad** [comment]
It'd be cool to enable that, and 'fail on error' option as well, just like the options for 'protractor-console-plugin'. Then this plugin would be superior both in terms of cross-compatibility and the looks.

**Comment by pmowrer** [comment]
@dinvlad Have a look at [protractor-fail-fast](https://github.com/Updater/protractor-fail-fast) for "fail on error" behavior.

Noted on the exclude behavior. I'll try to have a look at that soon.

**Comment by dinvlad** [comment]
Yes, the fail-fast plugin could help but unfortunately it seems to be targeted for Jasmine. I use protractor-cucumber-framework instead, so it cannot be applied (or I just don't know how). Thanks!

**Comment by pmowrer** [comment]
I see, that's too bad. I guess that firmly establishes that fail-fast behavior belongs outside of this library since there are many different test runners to account for.
